src/components/MultiLink/Legend.vue

```diff
@@ -58,6 +59,45 @@ export default Vue.extend({
     nodeColorScale() {
       return store.getters.nodeColorScale;
     },
+
+    columnTypes() {
+      const typeMapping: { [key: string]: string} = {};
+
+      if (store.getters.networkMetadata !== null) {
+        // eslint-disable-next-line @typescript-eslint/no-unused-vars
+        Object.entries(store.getters.networkMetadata).forEach(([tableName, metadata]) => {
+          (metadata as TableMetadata).table.columns.forEach((columnType) => {
+            typeMapping[columnType.key] = columnType.type;
+          });
+        });
+      }
+
+      return typeMapping;
+    },
+
+    cleanedNodeVariables(): Set<string> {
+      const cleanedVariables = new Set<string>();
+
+      (this.multiVariableList as Set<string>).forEach((variable) => {
+        if (this.columnTypes[variable] !== 'label') {
+          cleanedVariables.add(variable);
+        }
+      });
+
+      return cleanedVariables;
+    },
+
+    cleanedLinkVariables(): Set<string> {
+      const cleanedVariables = new Set<string>();
+
+      (this.linkVariableList as Set<string>).forEach((variable) => {
+        if (this.columnTypes[variable] !== 'label') {
+          cleanedVariables.add(variable);
+        }
+      });
+
+      return cleanedVariables;
+    },
```

Comment:
You may want to factor out the core of these two methods into a pure function.

Reply:
I had actually already done this before your review, I just forgot to push it. See a041160